extra/resources/SysInfo.in

```diff
@@ -84,7 +84,7 @@ Units:
 <content type="string" default="$OCF_RESKEY_pidfile" />
 </parameter>
 
-<parameter name="delay">
+<parameter name="delay" reloadable="1">
```

Comment:
This one is a little tricky since both the start and monitor actions calls SysInfoStats -> UpdateStat

The difference is that start uses -B with attrd_updater (which can update both the delay and the attribute value) while monitor uses -v (which only updates the attribute value). So, delay will not be reloadable since it can only change at start.

Alternatively, if we always used -B, delay would be reloadable. I'm not sure why we didn't do that to begin with; I suspect we were just being conservative when -B was introduced, but I don't see any problem with it.